ci-operator/config/openshift/splunk-forwarder-images/openshift-splunk-forwarder-images-master.yaml

```diff
@@ -0,0 +1,53 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
```

Comment:
Note to reviewers:

Yes, this is weird.

We [need](https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image) a `root` image that contains a `git` command, which apparently `ubi-minimal` does not. That image is only used to build `src`, which is what clones in the source repo, which is how we obtain the Dockerfiles etc. At that point the Dockerfiles are run `FROM base` -- i.e. the `ubi-minimal` above.

So TL;DR: The golang image doesn't figure in the splunk-(heavy)forwarder images at all. We just need to pull it (or something like it) in to make the pipeline work.

See [this slack thread](https://coreos.slack.com/archives/CBN38N3MW/p1621018598439200) for details.